Mathlib/CategoryTheory/Action/Monoidal.lean

```diff
@@ -42,30 +42,23 @@ instance instMonoidalCategory : MonoidalCategory (Action V G) :=
 /- Adding this solves `simpNF` linter report at `tensorUnit_ρ` -/
 @[simp]
 theorem tensorUnit_ρ' {g : G} :
-    @DFunLike.coe (G →* MonCat.of (End (𝟙_ V))) _ _ _ (𝟙_ (Action V G)).ρ.hom g = 𝟙 (𝟙_ V) := by
+    @DFunLike.coe (G →* End (𝟙_ V)) _ _ _ (𝟙_ (Action V G)).ρ g = 𝟙 (𝟙_ V) := by
   rfl
 
 @[simp]
 theorem tensorUnit_ρ {g : G} :
```

Comment:
Could you experiment the following: remove this lemma `tensorUnit_ρ` and rename the one above `tensorUnit_ρ'` as `tensorUnit_ρ` (and similarly for `tensor_ρ`).
Indeed, it seems the current `tensorUnit_ρ` is not a "good" simp lemma, because after we do `dsimp` on the LHS, it is no longer able to "prove itself" by `rw` or `simp` (because some invisible terms are simplified by `dsimp`).
If the experiment above works (on my computer, it does not break the file, but we should try the whole CI tests), then this would be a small improvement of the API.
(Ideally, we should have a tactic/meta-code "dsimp the LHS of my lemma" which would take care of such issues automatically, but nobody has implemented it yet.)

Reply:
Thanks, that worked!